spec/traits.dd

```diff
@@ -630,12 +630,35 @@ writeln(__traits(identifier, var));             // "var"
 
 $(H2 $(GNAME getAliasThis))
 
-    $(P Takes one argument, a symbol of aggregate type.
-        If the given aggregate type has $(D alias this), returns a list of
-        $(D alias this) names, by a tuple of $(D string)s.
-        Otherwise returns an empty tuple.
+    $(P Takes one argument, a symbol. If the symbol is of a type which has
+        $(D alias this), returns a list of $(D alias this) names, by a tuple
```

Comment:
names, by a tuple -> names as a tuple

Reply:
The name we use to call what `AliasSeq` represents was debated at length in the newsgroup a couple years ago and we agreed that we should call them "sequences" to distinguish them from tuple-like objects like the one defined in `std.typecons.Tuple`.

The difference is the following:
* Tuples are value-types holding other run-time specified values, whose types are fixed at compile-time, i.e. at run-time you can change value of the i-th element, but not its type. Nesting is respected, i.e. `Tuple!(int, Tuple!(Tuple!double)) != Tuple!(int, double)`.

* Sequences are lists of compile-time entities (which can include literal values, types, and symbols). They are immutable and they don't nest, i.e. `AliasSeq!(int, AliasSeq!(AliasSeq!double)) == AliasSeq!(int, double)`

So, in summary I recommend the following wording:

```suggestion
        Takes one argument, a type. If the type has `alias this` declaration(s),
        returns the sequence members (represented as `string`s) which were
        part of those declarationd. Otherwise returns an empty sequence.
```

Reply:
I have not adopted the suggested wording exactly, because to me "returns the sequence members which [...]" sounds like we are selecting some members of an existing sequence. Please review whether the new wording is acceptable.